src/crimson/osd/pg.cc

```diff
@@ -934,11 +934,8 @@ PG::submit_transaction(
     std::move(submitted),
     all_completed.then_interruptible(
       [this, last_complete=peering_state.get_info().last_complete, at_version]
-      (auto acked) {
-      for (const auto& peer : acked) {
-        peering_state.update_peer_last_complete_ondisk(
-          peer.shard, peer.last_complete_ondisk);
-      }
+      (auto acked_peers) {
+      logger().debug("{} acked_peers: {}", __func__, acked_peers);
```

Comment:
I'd *really* like to avoid adding any new instances of the old logging mechanism.  The problem with this is that it doesn't include the pg status prefix, so grepping for the PG will skip this line.  Can we use DEBUGDPP here instead?  We can convert the rest of the file at another time, I just want to avoid new ones.

Reply:
Um, this log has been removed in the latest update.

Reply:
Ah, I was looking at the commits, nvm then.